src/utils/logger.js

```diff
@@ -0,0 +1,32 @@
+require('dotenv').config();
+const moment = require('moment');
+const winston = require('winston');
+
+var config = winston.config;
+var logger = new (winston.Logger)({
+  transports: [
+    new (winston.transports.Console)({
+      timestamp: function() {
+        return moment().format("YYYY-MM-DD HH:mm:ss")
+      },
+      formatter: function(options) {
+        return `${options.timestamp()} ${__filename} ${config.colorize(options.level, options.level.toUpperCase())} ${(options.message ? options.message : '')} ${(options.meta && Object.keys(options.meta).length ? '\n\t'+ JSON.stringify(options.meta) : '' )}`;
+      }
+    }),
+    new (winston.transports.File)({
+      filename: `${__dirname}/../logs/${moment().format("YYYY-MM-DD")}.log`,
+      timestamp: function() {
+        return moment().format("YYYY-MM-DD HH:mm:ss")
+      }, 
+      formatter: function(options) {
+        return `${options.timestamp()} ${__filename} ${config.colorize(options.level, options.level.toUpperCase())} ${(options.message ? options.message : '')} ${(options.meta && Object.keys(options.meta).length ? '\n\t'+ JSON.stringify(options.meta) : '' )}`;
+      },
+      json: false,
+    })
+  ],
+  exitOnError: false
+});
+const loglvl = process.env.loglvl || 'info';
```

Comment:
1. `info` should read from config file
2. where is process.env.loglvl defined?

Reply:
process.env.loglvl is defined in the .env file